### PR TITLE
Update NuGet packages

### PR DESCRIPTION
**Pull request by bricelam**

### CODE REVIEW

**Review by smitpatel — APPROVED** [review]

File: src/EFCore.SqlServer/EFCore.SqlServer.csproj [hunk 1]
```diff
@@ -20,7 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Data.SqlClient" Version="2.1.4" />
+    <PackageReference Include="Microsoft.Data.SqlClient" Version="4.0.0-preview3.21293.2" />
```
Comment (roji) [inline src/EFCore.SqlServer/EFCore.SqlServer.csproj:23]: SqlClient 4.0.0 just got released, may as well update to that...

Comment (roji) [inline src/EFCore.SqlServer/EFCore.SqlServer.csproj:23]: Just released:

```suggestion
    <PackageReference Include="Microsoft.Data.SqlClient" Version="4.0.0" />
```

**Comment by bricelam** [comment]
We need to mirror all these packages. I just need to find the gigantic closure for IdentityServer

**Comment by bricelam** [comment]
@wtgodbe Can we mirror the following packages?

Package | Version
--- | ---
Grpc.AspNetCore.Server | 2.41.0
Grpc.AspNetCore.Server.ClientFactory | 2.41.0
Grpc.AspNetCore | 2.41.0
Grpc.Net.Client | 2.41.0
Grpc.Net.ClientFactory | 2.41.0
Grpc.Net.Common | 2.41.0
Grpc.Tools | 2.41.0
Humanizer.Core | 2.13.14
IdentityServer4.EntityFramework | 4.1.2
IdentityServer4.EntityFramework.Storage | 4.1.2
IdentityServer4.Storage | 4.1.2
IdentityServer4 | 4.1.2
NetTopologySuite | 2.4.0

**Comment by wtgodbe** [comment]
Done - for future reference, I don't need the versions. The script we use mirrors all versions that aren't already on the AzDO feed

**Pull request merged**